server/src/test/java/org/spine3/testdata/Sample.java

```diff
@@ -113,12 +175,16 @@ private static void checkClass(Class<? extends Message> clazz) {
         checkNotNull(clazz);
         // Support only generated protobuf messages
         checkArgument(clazz.isAssignableFrom(GeneratedMessageV3.class));
```

Comment:
Assuming that we perform the check, we need to propagate this requirement into method signatures here and above the call chain. Otherwise, there's no way to know it.